recipes/dnmtools/meta.yaml

```diff
@@ -23,9 +23,11 @@ requirements:
     - llvm-openmp  # [osx]
     - libgomp      # [linux]
   host:
+    - zlib
     - htslib
     - gsl
   run:
+    - zlib
```

Comment:
According to https://bioconda.github.io/faqs.html?highlight=zlib#how-does-global-pinning-work this is not really needed!
```suggestion
```